src/utils/commands.json

```diff
@@ -38,6 +38,11 @@
             "command2": "bash",
             "description": "This allows you to execute an interactive bash shell on a running container."
         },
+        "Access pgsql database": {
```

Comment:
Sorry we didn't do a good job on documenting on how this json file work.
So the `command` field is for anything before the `option` field, and the `command2` field is used when there is a second part of the command that runs after the `option` field. So in this case it would look something like the code below.
(I'd also be very helpful if you can document this and add it to the top of this json file / to the README file for this repo please 🙇 )
```
"Access pgsql database": {
             "command": "docker container exec -it",
             "option": "$CONTAINER",
             "command2": "psql -U postgres sourcegraph"
             "description": "This allows you to access the main database and run psql commands on a running container."
         },
```

Reply:
@abeatrix I finally got around to this 🤣 Please see my updates to the command and README file. [I did not add the styling to the json file because apparently comments ](https://stackoverflow.com/questions/244777/can-comments-be-used-in-json)aren't common practice 🤷‍♀️